intake/source/cache.py

```diff
@@ -0,0 +1,228 @@
+from datetime import datetime
+from hashlib import md5
+from pathlib import Path
+
+import json
+import logging
+import os
+import shutil
+
+from intake.config import conf
+
+logger = logging.getLogger('intake')
+
+
+def disable_caching(state):
+    os.environ['INTAKE_DISABLE_CACHING'] = '1' if state else '0'
```

Comment:
Funny to do this via the environment - I would have had a module-global variable, which takes its initial value from the environment or config (the former can override the latter).

Reply:
I just made this a conf parameter initialized by the env var.